src/reporting/parameter.ts

```diff
@@ -142,6 +142,16 @@ export function createStackParametersQuickSight(scope: Construct, paramGroups?:
     default: 'Parameter Key Name',
   };
 
+  const redshiftIAMRoleParam = new CfnParameter(scope, 'RedshiftIAMRoleParam', {
+    description: 'The ARN of IAM role used by Redshift Data API.',
+    type: 'String',
+    default: '',
```

Comment:
Is it a mandatory parameter? If so, let's remove default value and update allowedPattern.